### PR TITLE
update Performance index page and restructure

### LINKED ISSUE #3358: Update Performance Monitoring index page
## Issue

This an issue with:
Performance (product)
https://docs.sentry.io/product/performance/ 

## Description

Performance Monitoring page doesn't follow the current standard for product index pages. It currently includes a lot of information describing actions users might take on the page.

### Suggested Solution

Update this page to provide a high-level description the feature in terms of what it is/what it does for the customer. 

- [x] Draft new structure
- [x] Create new child page for Display Selection and other filter/display content to be moved from index page
- [x] Update content of index page including adding links to new child page content
- [ ]

### PR DESCRIPTION
**Pull request by imatwawana**
Improve Performance index page and move content to new Filters & Display page
Reorder child pages of Performance
Fix redirects

### CODE REVIEW

File: src/docs/product/performance/filters-display.mdx [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+- [Transactions Per Minute (TPM)](/product/performance/metrics/#throughput-total-tpm-tps)
+- [Failure Rate](/product/performance/metrics/#failure-rate)
+- [p50 Duration](/product/performance/metrics/#p50-threshold)
+- [p95 Duration](/product/performance/metrics/#p95-threshold)
+- [p99 Duration](/product/performance/metrics/#p99-threshold)
+
+## Transaction Table
+
+The table of transactions provides you with a list of the most frequent transactions based on the current page filters and display options. By default, the table is sorted by transactions per minute, with the most frequent transactions displayed first. The table may be filtered further based on global header and search condition filters.
+The [Display Filter](/product/performance/display-filters/#display-filter) will automatically set what appears in the table columns. Here are a few of the common columns shown in the tables.
```
Comment (imatwawana) [inline src/docs/product/performance/filters-display.mdx:59]:
```suggestion
The [Display Filter](/product/performance/filters-display/#display-filter) will automatically set what appears in the table columns. Here are a few of the common columns shown in the tables.
```

File: src/docs/product/performance/distributed-tracing.mdx [hunk 2]
```diff
@@ -1,9 +1,8 @@
 ---
 title: Distributed Tracing
-sidebar_order: 3
+sidebar_order: 60
 redirect_from:
-  - /performance/distributed-tracing/
-  - /performance-monitoring/distributed-tracing/
+  - /product/performance-monitoring/distributed-tracing/
   - /product/sentry-basics/guides/error-tracing/
 description: "Learn more about distributed tracing and how Sentry tracks your software performance, measures metrics like throughput and latency, and displays the impact of errors across multiple systems."
```
Comment (PeloWriter) [inline src/docs/product/performance/distributed-tracing.mdx:7]:
```suggestion
description: "Learn more about distributed tracing and how Sentry tracks your software performance, measures throughput and latency, and displays the impact of errors across multiple systems."
```

File: src/docs/product/performance/event-detail.mdx [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 title: "Event Detail"
-sidebar_order: 6
+sidebar_order: 70
 description: "Learn more about Performance Event Detail and how you can drill all the way down into a span for a single transaction and traverse multiple directions. This will accelerate your ability to debug slow HTTP requests, database queries, identify associated errors, and root out other bottlenecks."
```
Comment (PeloWriter) [inline src/docs/product/performance/event-detail.mdx:4]:
```suggestion
description: "Learn more about how you can drill down into a span for a single transaction and traverse multiple directions to debug slow HTTP requests, database queries, identify associated errors, and root out other bottlenecks."
```

File: src/docs/product/performance/filters-display.mdx [hunk 4]
```diff
@@ -0,0 +1,72 @@
+---
+title: "Filters & Display"
+sidebar_order: 20
+redirect_from:
+  - /product/performance/display/
+description: "Filter the information displayed on the Performance page using search or various page-level display filters."
```
Comment (PeloWriter) [inline src/docs/product/performance/filters-display.mdx:6]:
```suggestion
description: "Manage the information on the Performance page using search or page-level display filters to quickly identify performance issues."
```

File: src/docs/product/performance/filters-display.mdx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+---
+
+There are several ways you can filter the information displayed on the **Performance** page, by using search or various page-level display filters.
```
Comment (PeloWriter) [inline src/docs/product/performance/filters-display.mdx:9]:
```suggestion
You can filter the information displayed on the **Performance** page by searching or using page-level display filters.
```

File: src/docs/product/performance/filters-display.mdx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+## Transaction and Trend Views
+
+Using the “By Transaction/By Trend” toggle, you can set the overall display of the page. By default, the page is set to view transactions and the different ways that you can filter and set the display of transactions is described in the rest of this page. Learn more about viewing transactions by trend in the [Trends View documentation](/product/performance/trends/).
```
Comment (PeloWriter) [inline src/docs/product/performance/filters-display.mdx:13]:
```suggestion
The page default is to view transactions, but you can use the “By Transaction/By Trend” toggle to change the overall display of the page. The different ways you can filter and set the display of transactions is described fully on this page. To learn more about viewing transactions by trend, see our [Trends View documentation](/product/performance/trends/).
```

File: src/docs/product/performance/filters-display.mdx [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+
+## Search Condition Filters
+
+The search bar operates similarly to the one in the Discover [Query Builder](/product/discover-queries/query-builder/#filter-by-search-conditions). For example, you can refine your transaction search to a specific release by using `release:` as a key field and assigning the version as a value.
+
+## Display Filter
+
+The "Display" filter can be found next to the search bar. This  automatically selects a set of [metrics](/product/performance/metrics/) and table columns based on the platform of your project. For example, a JavaScript project defaults to a Frontend (Pageload) display. You can change the selected display, but some modes may not be applicable to the platform.
```
Comment (PeloWriter) [inline src/docs/product/performance/filters-display.mdx:21]:
```suggestion
The "Display" filter is next to the search bar. It automatically selects a set of [metrics](/product/performance/metrics/) and table columns based on the platform of your project. For example, a JavaScript project defaults to a Frontend (Pageload) display. You can change the selected display, though note that some modes are not applicable to the platform.
```

File: src/docs/product/performance/filters-display.mdx [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+### All Transactions
+
+When "All Transactions" is selected, the table includes all frontend and backend transactions.
```
Comment (PeloWriter) [inline src/docs/product/performance/filters-display.mdx:25]:
```suggestion
When you select "All Transactions", the table includes all frontend and backend transactions.
```

File: src/docs/product/performance/filters-display.mdx [hunk 9]
```diff
@@ -0,0 +1,72 @@
...
+
+### Backend
+
+"Backend" puts an emphasis on duration, throughput, failure rate, and Apdex. Note that if you've selected a frontend project, this mode also displays all frontend transactions.
+
+![Performance Homepage with Display: Backend](02_performance_landing.png)
+
+## Graphs
+
+You can compare graphs side by side to find any relevant correlations. For example, if an Apdex score dips significantly during a given time period, you'll be able to visualize whether throughput spiked in the same time interval in a "Transactions per Minute" graph. The global header and search conditions will also allow you to filter these graphs even more. The display options are listed below:
```
Comment (PeloWriter) [inline src/docs/product/performance/filters-display.mdx:47]:
```suggestion
You can compare graphs side by side to find any relevant correlations. For example, if an Apdex score dips significantly during a given time period, you'll be able to visualize whether throughput spiked in the same time interval in a "Transactions per Minute" graph. The global header and search conditions will also allow you to further filter these graphs:
```

File: src/docs/product/performance/filters-display.mdx [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+- [Apdex](/product/performance/metrics/#apdex)
+- [Transactions Per Minute (TPM)](/product/performance/metrics/#throughput-total-tpm-tps)
+- [Failure Rate](/product/performance/metrics/#failure-rate)
+- [p50 Duration](/product/performance/metrics/#p50-threshold)
+- [p95 Duration](/product/performance/metrics/#p95-threshold)
+- [p99 Duration](/product/performance/metrics/#p99-threshold)
+
+## Transaction Table
+
+The table of transactions provides you with a list of the most frequent transactions based on the current page filters and display options. By default, the table is sorted by transactions per minute, with the most frequent transactions displayed first. The table may be filtered further based on global header and search condition filters.
```
Comment (PeloWriter) [inline src/docs/product/performance/filters-display.mdx:58]:
```suggestion
The table of transactions provides a list of the most frequent transactions based on the current page filters and display options. By default, the table is sorted by transactions per minute, with the most frequent transactions displayed first. The table may be filtered further based on global header and search condition filters.
```

File: src/docs/product/performance/filters-display.mdx [hunk 11]
```diff
@@ -0,0 +1,72 @@
...
+The [Display Filter](#display-filter) will automatically set what appears in the table columns. Here are a few of the common columns shown in the tables.
```
Comment (PeloWriter) [inline src/docs/product/performance/filters-display.mdx:59]:
```suggestion
The [Display Filter](#display-filter) controls what displays in the table columns, such as:
```

File: src/docs/product/performance/index.mdx [hunk 12]
```diff
@@ -2,69 +2,44 @@
...
 
-### All Transactions
+The Performance page provides you with several filter and display options so that you can focus on the performance data you want. Search conditions help you find the transactions you want to investigate. You can also control what kind of information is visualized in the [graphs](/product/performance/filters-display/#graphs) or which transactions appear first in the [transactions table](/product/performance/filters-display/#transaction-table).
 
-All Transactions will be selected by default. The table will include all frontend and backend transactions.
-![Performance Homepage with Display: All](00_performance_landing.png)
+## Analyze Performance Trends
 
-### Frontend (Pageload)
+The [trends view](/product/performance/trends/) surfaces transactions that have had significant changes in their performance over time. This view is ideal for providing you with insights when you have transactions with large counts.
```
Comment (PeloWriter) [inline src/docs/product/performance/index.mdx:25]:
```suggestion
The [trends view](/product/performance/trends/) surfaces transactions that have had significant changes in their performance over time. This view is ideal for providing insights when you have transactions with large counts.
```

**Review by PeloWriter — APPROVED** [review]
Edits to smooth. Looks good.

**Review by jeremybourque — APPROVED** [review]
LGTM

File: src/docs/product/performance/filters-display.mdx [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+
+The page default is to view transactions, but you can use the “By Transaction/By Trend” toggle to change the overall display of the page. The different ways you can filter and set the display of transactions is described fully on this page. To learn more about viewing transactions by trend, see our [Trends View documentation](/product/performance/trends/).
+
+## Search Condition Filters
+
+The search bar operates similarly to the one in the Discover [Query Builder](/product/discover-queries/query-builder/#filter-by-search-conditions). For example, you can refine your transaction search to a specific release by using `release:` as a key field and assigning the version as a value.
+
+## Display Filter
+
+The "Display" filter is next to the search bar. It automatically selects a set of [metrics](/product/performance/metrics/) and table columns based on the platform of your project. For example, a JavaScript project defaults to a Frontend (Pageload) display. You can change the selected display, though note that some modes are not applicable to the platform.
```
Comment (silent1mezzo) [inline src/docs/product/performance/filters-display.mdx:21]: Could also be `not applicable for the selected platform`. The current wording just feels off.
```suggestion
The "Display" filter is next to the search bar. It automatically selects a set of [metrics](/product/performance/metrics/) and table columns based on the platform of your project. For example, a JavaScript project defaults to a Frontend (Pageload) display. You can change the selected display, though note that some modes are not applicable to all platforms.
```

**Review by silent1mezzo — APPROVED** [review]
One minor suggestion but LGTM 👍

**Pull request merged**